CHANGELOG.txt

```diff
@@ -1,3 +1,10 @@
+Improvements
+""""""""""""
+
+- The Samba hook now uses the `smbprotocol` library, exposing a much
+  bigger set of functionality and supporting SMB2/3 protocols.
```

Comment:
This is in wrong file -- it should be at https://github.com/apache/airflow/blob/main/airflow/providers/samba/CHANGELOG.rst

Reply:
@kaxil thanks – I have removed this entry again. I had actually already written a more detailed changelog entry in the provider changelog.